tests/fiaas_deploy_daemon/deployer/kubernetes/deployment/test_datadog.py

```diff
@@ -57,7 +58,9 @@ def test_adds_env_when_enabled(self, datadog, app_spec, deployment, best_effort_
         ("rincewind", "discworld")
     ))
     def test_adds_container_when_enabled(self, datadog, app_spec, deployment, best_effort_required, name, namespace):
-        app_spec = app_spec._replace(datadog=True, name=name, namespace=namespace)
+        datadog_spec = app_spec.datadog._replace(enabled=True, tags=dict())
```

Comment:
I think it would be nice to have a unit test that covers the code path that handles tags added in the fiaas config too